irods/test/data_obj_test.py

```diff
@@ -2362,6 +2362,33 @@ def _update(n):
 
         self.assertEqual(pbar.percent_done(), 100)
 
+    def test_replica_truncate__issue_534(self):
```

Comment:
Consider adding a test that proves increasing the size of a replica works as intended.

Reply:
I would add that we should get one test with an error response to see what that looks like as well (e.g. a negative size or specifying a resource which does not have a replica of the target data object). Just so we exercise the exceptions.

Reply:
Agreed.

Reply:
We get SYS_REPLICA_INACCESSIBLE (168000) if we target a resource not housing a replica of the data obj in question.  That seems good to me. Shall we have a test asserting that particular type of exception / error?

Reply:
Sounds good to me. The important part is that we are exercising the error mechanism of the PRC method, but we can /should also assert that the error code is correct, yes.

Reply:
And `UNIX_FILE_TRUNCATE_ERR` with OS errno 22 when a negative size is fed in ....

Reply:
Done! Wrote test including a reasonable range of errors one might see appear in this API / endpoint....